linode/resource_linode_object_storage_bucket.go

```diff
@@ -129,6 +162,28 @@ func resourceLinodeObjectStorageBucketDelete(d *schema.ResourceData, meta interf
 	return nil
 }
 
+func updateLinodeObjectStorageBucketAccess(d *schema.ResourceData, client linodego.Client) error {
+	cluster := d.Get("cluster").(string)
+	label := d.Get("label").(string)
+
+	updateOpts := linodego.ObjectStorageBucketUpdateAccessOptions{}
+	if d.HasChange("acl") {
+		updateOpts.ACL = linodego.ObjectStorageACL(d.Get("acl").(string))
+	}
+
+	if d.HasChange("cors_enabled") {
+		newCorsBool := d.Get("cors_enabled").(bool)
+		updateOpts.CorsEnabled = &newCorsBool
+	}
+
+	err := client.UpdateObjectStorageBucketAccess(context.Background(), cluster, label, updateOpts)
+	if err != nil {
+		fmt.Errorf("Error updating Linode ObjectStorageBucket access: %s", err)
```

Comment:
I would avoid using the word `error` in error messages. Also, error strings shouldn't be capitalized. https://github.com/golang/lint/blob/738671d3881b9731cc63024d5d88cf28db875626/lint.go#L1178